docker-compose.local.yml

```diff
@@ -15,15 +11,7 @@ services:
       - './:/home/app/'
       - '/home/app/node_modules'
       - '/home/app/broker-cli/node_modules'
+    environment:
+      - NODE_ENV=development
```

Comment:
hmm, so does this mean that hosted regtest is considered a "production" environment? I don't think I agree with that characterization.

Reply:
@treygriffith we would consider hosted regtest to be `staging`. development is local

Reply:
I would not consider hosted regtest production either

Reply:
A background on this flag, this is a solution for broker configuration so that we can have regtest on a staging and local environment. I stuck to environment conventions with `NODE_ENV` cause it was easier than creating a separate `BROKER_ENVIRONMENT` flag that controlled this